### ISSUE TITLE
Working directory / export folder workflow

### ISSUE THREAD

**Issue by mifi** [mifi/lossless-cut#2101]
Suggsted by email:

Drop-down menu to change the export folder quickly with options:
- Same as source
- Recent/last ones used
- List of favourite folder locations?

![46KAOko](https://github.com/user-attachments/assets/58013ecc-ec68-48f3-b514-7948b50b3d6a)

**Comment by mifi** [comment]
moved to #2115 - closing to reduce issue count

**Comment by sardinhunt** [comment]
I agree that working on a fast SSD helps a lot—personally, I manage by organizing and moving files onto the SSD before editing and keeping my main drive spacious. But the lack of output folder favorites (as discussed in [#2101](https://github.com/mifi/lossless-cut/issues/2101)) remains a real workflow issue.

I currently run a helper script that watches the LosslessCut folder and asks where to move files. I’m now building a “don’t ask again” toggle so it just moves clips to a default folder unless changed—basically solving this with extra steps.

It’d be much cleaner if LosslessCut supported a favorite output path natively. If that helps, I’d be happy to offer €100 to help prioritize this feature in the next 2–3 months—if that’s something you're open to. Maybe it could encourage more incentive-driven donations too.
